src/nlua_shiplog.c

```diff
@@ -0,0 +1,140 @@
+/*
+ * See Licensing and Copyright notice in naev.h
+ */
+
+/**
+ * @file nlua_shiplog.c
+ *
+ * @brief Handles the shiplog Lua bindings.
+ */
```

Comment:
You could use this comment section to show a full example of log usage in between `@code` and `@endcode` tags. Take a look at the other modules for an example.

🕵️

Reply:
Example added.